tests/entities/conftest.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import random
+import uuid
+
+from tests.helper_functions import random_str, random_int
+
+from mlflow.entities import RunInfo, LifecycleStage, RunStatus
+from mlflow.entities import Metric, RunData, Param, RunTag
+from mlflow.utils.time_utils import get_current_time_millis
+
+
+@pytest.fixture(scope="module")
```

Comment:
Does this fixture really have to be module-scoped?

Reply:
No, changed it to function.